.github/workflows/tag-release-from-main.yml

```
@@ -0,0 +1,58 @@
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v5
+        with:
+          fetch-depth: 0
+
+      - name: Parse version from commit message
+        id: version
+        shell: bash
+        run: |
```

actionlint (1.7.12)

[error] "github.event.head_commit.message" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/reference/security/secure-use#good-practices-for-mitigating-script-injection-attacks for more details

(expression)